UsersGitHub/UsersGitHub/ViewModel/LoginPageViewModel.cs

```diff
@@ -45,11 +47,46 @@ public LoginPageViewModel()
         }
 
         private async void GoToUserReposPage()
-        {          
-            BlobCache.ApplicationName = "UsersGitHub";
-            var user = new User { Login = UserLogin };
+        {
+            var name = await GetUserInfo(UserLogin);
+            var repositories = await GetUserRepositories(UserLogin);
+            var user = new User
+            {
+                UserName = name,
+                Repositories = repositories
+            };
             await BlobCache.UserAccount.InsertObject(UserLogin, user);
             Application.Current.MainPage = new UsersReposPage();
         }
+
+        private async Task<string> GetUserInfo(string userName)
+        {
+            var httpClient = new HttpClient
```

Comment:
calls to API shouldn't be done in viewmodel
please, extract them to a separate service